### PR TITLE
Create ui-c8y-1018-503-130-nightly-failure-data-and-control-team-device-protocol-loriot.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dmajsag

Ticket: [DM-4173](https://cumulocity.atlassian.net/browse/DM-4173)
Original PR: [7516](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7516)

[DM-4173]: https://cumulocity.atlassian.net/browse/DM-4173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1018-503-130-nightly-failure-data-and-control-team-device-protocol-loriot.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Nightly failure dataAndControlTeam deviceProtocolLoriot
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-130-nightly-failure-data-and-control-team-device-protocol-loriot.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix nightly failure in dataAndControlTeam deviceProtocolLoriot test
```

File: content/change-logs/device-management/ui-c8y-1018-503-130-nightly-failure-data-and-control-team-device-protocol-loriot.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4173
+version: 1018.503.130
+---
+Nightly failure dataAndControlTeam deviceProtocolLoriot
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-130-nightly-failure-data-and-control-team-device-protocol-loriot.md:17]: The Claude AI suggested to use the following body:
```suggestion
The dataAndControlTeam deviceProtocolLoriot test was failing in the nightly test runs. The test has been fixed to properly handle the test scenario and prevent the failure. The fix ensures that the deviceProtocolLoriot functionality works as expected and the nightly test runs complete successfully without any failures.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@l2p- @dmajsag I assume that we can close this PR as this change should not get a change log?

**Pull request closed without merging**